k8s.io/configmap-nginx.yaml

```diff
@@ -144,7 +144,7 @@ data:
         listen 80;
         listen 443 ssl;
 
-        rewrite ^/(.*)?$    https://github.com/kubernetes/kubernetes/releases/tag/$1 redirect;
+        rewrite ^/(.*)?$    https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md redirect;
```

Comment:
I don't think this is quite right... something like https://changelog.k8s.io/v1.11.2 now doesn't redirect as usefully.

I agree that with an empty request URI this is better. maybe have multiple cases?

Reply:
You could do both though, as @ixdy is suggesting. If you have the rewrite case of `^/$` then if there is no path, it will redirect to a different location. Just put that ahead of the per-version rewrite rule